runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/Environments.java

```diff
@@ -79,13 +79,16 @@
 
   private static final String dockerContainerImageOption = "docker_container_image";
   private static final String externalServiceAddressOption = "external_service_address";
+  private static final String externalServiceExecutableOption = "external_service_executable";
```

Comment:
External workers should be completely decoupled from the runner. If we introduce `external_service_executable` as a pipeline option, we have to add it to the contract between the runner and the worker.

The configuration of the workers should be left to the worker pool wherever possible. So instead of a pipeline option, we'd add command line arguments in the worker pool's main method. WDYT?

Reply:
This makes sense, the runner should be ignorant of how external worker executes. I do see this is the pattern that Python external worker follows.